src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -313,5 +323,117 @@ public static AnnotatedCodeLocationImportanceVersionOne CreateAnnotatedCodeLocat
                     return AnnotatedCodeLocationImportanceVersionOne.Important;
             }
         }
+
+        public interface IBlameHunk
+        {
+            string Name { get; set; }
+
+            string Email { get; set; }
+
+            string CommitSha { get; set; }
+
+            int LineCount { get; }
+
+            int FinalStartLineNumber { get; }
+
+            bool ContainsLine(int line);
+        }
+
+        private class BlameHunk : IBlameHunk
+        {
+            private readonly string _Name;
```

Comment:
>_Name [](http://example.com/codeflow?start=36&length=5)

can you use lower-case? _name, _email, etc.

Reply:
Fixed.